src/add/index.js

```diff
@@ -16,21 +16,36 @@ module.exports = configure(({ ky }) => {
     if (options.chunker) searchParams.set('chunker', options.chunker)
     if (options.cidVersion) searchParams.set('cid-version', options.cidVersion)
     if (options.cidBase) searchParams.set('cid-base', options.cidBase)
-    if (options.enableShardingExperiment != null) searchParams.set('enable-sharding-experiment', options.enableShardingExperiment)
+    if (options.enableShardingExperiment != null) {
```

Comment:
Were these changes necessary?  I personally like curly brackets around if statement bodies, even one liners, but this has made everything inconsistent and is outside the scope of the intended change.

Reply:
No, they weren't necessary. They happened due to the default code styling settings I have globally in VS Code. If this style is important and gates PR merging, though, we should modify the linter settings to throw a warning or an error with this style. Ideally, we would get `aegir lint --fix` to automatically resolve such warnings/errors too, which may happen out of the box when we enable the warning. I'll open an issue for this.

Reply:
Just opened the issue https://github.com/ipfs/js-ipfs-http-client/issues/1225

Reply:
I've disabled the `Prettier` and `JavaScript Standard Style` extensions for this workspace locally in VS Code so that automatic linting changes won't make it into a PR against this repo next time, but I still think the linting settings change is worth making to make it simpler for others to make PRs in the future.